samples/hybrid-connections/java/simple-http-demo/README.md

```diff
@@ -0,0 +1,100 @@
+# 'Simple' HTTP Hybrid Connection Sample 
+
+First, as described in the description [Here](https://github.com/bainian12345/azure-relay/tree/java-samples/samples/hybrid-connections/java), you will need to retrieve different parameters from the enviroment variable `RELAY_CONNECTION_STRING` for connection and authentication purposes:
```

Comment:
>bainian12345 [](start = 65, length = 12)

Confirm which fork (Azure vs. bainian12345) and branch should appear in the link here.